pandas/util/clipboard.py

```diff
@@ -138,7 +138,7 @@ def _copyGtk(text):
 
 
 def _pasteQt():
-    return str(cb.text())
+    return unicode(cb.text())
```

Comment:
yeah, i created this PR for testing purposes and all clipboard tests are passing. So i can make the change suggested in the original PR